app/recordtransfer/views/transfer.py

```diff
@@ -195,7 +242,19 @@ def post(self, request: HttpRequest, *args, **kwargs) -> HttpResponse:
 
         try:
             self.save_form_data(request)
-            messages.success(request, gettext("Transfer saved successfully."))
+            message = gettext("Transfer saved successfully.")
+            if (
+                expires_at := self.in_progress_submission.upload_session_expires_at
+                if self.in_progress_submission.upload_session
```

Comment:
`upload_session_expires_at` already checks if the upload session exists

```suggestion
                if self.in_progress_submission
```